app/commands/git/sync_pull_requests.rb

```diff
@@ -0,0 +1,163 @@
+module Git
```

Comment:
If we only intend to do this once as part of ETL, we should probably add a `Temp` module inside `Git` and namespace it in that. Then we can delete it as part of the deletion of any other temp code. If you think this might/will persist beyond ETL, then leave it as it is.